cmd/gwctl/subcommand/gwctl.go

```diff
@@ -114,3 +115,54 @@ func (o *stateGetOptions) run() error {
 	fmt.Println(string(sJSON))
 	return nil
 }
+
+type allGetOptions struct {
+	myID string
+}
+
+// AllGetCmd - get all gateways objects.
+func AllGetCmd() *cobra.Command {
+	o := allGetOptions{}
+	cmd := &cobra.Command{
+		Use:   "all",
```

Comment:
`gwctl get config-dump` ?

Reply:
I thought to do it similar to kubectl get all